### ISSUE TITLE
UniFace loss

### ISSUE THREAD

**Issue by ericyq** [HaiyuWu/SOTA-Face-Recognition-Train-and-Test#8]
line 269, in losses.py
one_hot = torch.zeros((labels[index].size(0), kwargs['out_features']), dtype=torch.bool)
here, is it  id class number to 'out_features'  ?

**Comment by HaiyuWu** [comment]
Based on the code in partial_fc.py, it should be the number of classes that are assigned to the current GPU.
https://github.com/HaiyuWu/SOTA-Face-Recognition-Train-and-Test/blob/026450b99486539a141446435ba2b6f744246cb6/model/partial_fc.py#L168

So, the answer is yes, but a part of id class numbers. I adapt it to fit partial-fc training.
